app/controllers/api/service_templates_controller.rb

```diff
@@ -4,5 +4,51 @@ class ServiceTemplatesController < BaseController
     include Subcollections::Tags
     include Subcollections::ResourceActions
     include Subcollections::ServiceRequests
+
+    def create_resource(_type, _id, data)
+      # Temporarily only supporting atomic.
+      raise 'Service Type composite not supported' if data['service_type'] == 'composite'
+      create_atomic(data).tap(&:save!)
+    rescue => err
+      raise BadRequestError, "Could not create Service Template - #{err}"
+    end
+
+    private
+
+    def create_atomic(data)
+      service_template = ServiceTemplate.new(data.except('request_info'))
+      if data.key?('request_info')
+        service_template.add_resource(create_service_template_request(data['request_info']))
```

Comment:
@bzwei exactly why I wanted this feedback 😃  is there a place in the code already doing this where I can follow along make it work for the REST api? Didn't quite see it in the existing controller...